Artesian/Artesian.SDK/Dto/UpsertCurveData/UpsertCurveData.cs

```diff
@@ -77,17 +77,22 @@ public UpsertCurveData(MarketDataIdentifier id, LocalDateTime version )
         [MessagePack.Key(5)]
         public IDictionary<LocalDateTime, double?> Rows { get; set; }
 
+        /// <summary>
+        /// The timeserie data in OriginalTimezone or, when Hourly, must be UTC.
+        /// </summary>
+        [MessagePack.Key(6)]
+        public IDictionary<LocalDateTime, AuctionBids> AuctionRows { get; set; }
 
         /// <summary>
         /// Flag to choose between synchronous and asynchronous command execution
         /// </summary>
-        [MessagePack.Key(6)]
+        [MessagePack.Key(7)]
```

Comment:
Changing Key number makes current version not compatible with older version.

I mean, that I'm sure server didn't do so else all writers would be failing in TEST as the client (this SDK prev this change) would send as position 6 a bool and the server would now expect a IDictionary.

I'm quite sure Alessandro didn't do such thing. Please verify that Key number matches with server-side.